feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -254,6 +257,33 @@ func (tc *testCase) transferOS(ctx context.Context, t *testing.T, standby bool)
 	}
 }
 
+func (tc *testCase) isSupervisorReady(ctx context.Context, t *testing.T) error {
+	var counter int8
+	const maxRetries = 15
+	for i := int8(1); i < maxRetries; i++ {
```

Comment:
do we need to specifically call out int8? your `maxRetries` variable didn't call out type anyway.

Reply:
No, we don't actually need to call out int8 here, I'll change everything to int.